src/layers/learning/fully_connected.cu

```diff
@@ -1,145 +0,0 @@
-////////////////////////////////////////////////////////////////////////////////
-// Copyright (c) 2014-2016, Lawrence Livermore National Security, LLC.
-// Produced at the Lawrence Livermore National Laboratory.
-// Written by the LBANN Research Team (B. Van Essen, et al.) listed in
-// the CONTRIBUTORS file. <lbann-dev@llnl.gov>
-//
-// LLNL-CODE-697807.
-// All rights reserved.
-//
-// This file is part of LBANN: Livermore Big Artificial Neural Network
-// Toolkit. For details, see http://software.llnl.gov/LBANN or
-// https://github.com/LLNL/LBANN.
-//
-// Licensed under the Apache License, Version 2.0 (the "Licensee"); you
-// may not use this file except in compliance with the License.  You may
-// obtain a copy of the License at:
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
-// implied. See the License for the specific language governing
-// permissions and limitations under the license.
-//
-// fully_connected_cuda.cu - GPU helper routines for fully connected layer
-////////////////////////////////////////////////////////////////////////////////
-
-#include "lbann/layers/learning/fully_connected_cuda.hpp"
-#include <cassert>
-
-namespace lbann {
-namespace fully_connected_cuda {
-
-__global__ void row_sum_kernel(El::Int h, El::Int w,
-                               const DataType *m,
-                               DataType factor,
-                               DataType *v,
-                               bool accum) {
-  int tid = blockIdx.x * blockDim.x + threadIdx.x;
-  if (tid > h) return;
-  DataType sum = 0;
-  for (int i = 0; i < w; ++i) {
-    sum += m[tid + i * h];
-  }
-  sum *= factor;
-  if (accum) sum += v[tid];
-  v[tid] = sum;
-}
-
-
-
-void row_sum(cudnn::cudnn_manager &cudnn,
```

Comment:
Do we no longer need this code?

Reply:
I've replace our custom CUDA kernels with cuBLAS calls.